products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,274 @@
+
+```opencypher
+MATCH (m:Team)<-[:TRANSFERRED_TO]-(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE m.name="FC Barcelona"
+WITH * ORDER BY p.name
+WITH DISTINCT p
+RETURN p.position as position, COUNT(p) AS position_count, collect(p.name) as playerNames
+ORDER BY position_count DESC
+```
+
+5) Football has seen a lot of rivalries develop between clubs during its rich and long history,
+some of which can be traced back more than a century. Now one of the most famous ones is between
+FC Barcelona and Real Madrid. Let's find out were there any transfers between FC Barcelona and Real Madrid.
+
+```opencypher
+MATCH path_teams = (m:Team )-[:TRANSFERRED_FROM]-(t:Transfer)-[:TRANSFERRED_TO]-(n:Team)
+WHERE (m.name="FC Barcelona" AND n.name="Real Madrid") OR (m.name="Real Madrid" AND n.name="FC Barcelona")
+MATCH path_player = (t)<-[:TRANSFERRED_IN]-(p:Player)RETURN path_teams,path_player
+```
+
+
+6) FC Barcelona is one of the biggest clubs in the world. Players often want to stay there as long as possible.
+But what about those players who didn't fit in well? Where do they go?
+
+```opencypher
+MATCH (m:Team )-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player), (t)-[:TRANSFERRED_TO]->(n:Team)
+WHERE m.name="FC Barcelona"
+RETURN n.name as team_name, collect(p.name) as names, COUNT(p) AS number_of_players
+ORDER BY number_of_players DESC
+```
+
+
+7) What are the teams that most players went to in season 2003/2004? The results may surprise you. 
+
+```opencypher 
+MATCH (s:Season {name:"2003/2004"})<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player)
+MATCH (t)-[:TRANSFERRED_TO]->(n:Team)
+WITH DISTINCT player, n
+RETURN n.name as team_name, COUNT(player) AS number_of_players,collect(player) as names
+ORDER BY number_of_players DESC, team_name
+LIMIT 20;
+```
+
+8)In great teams, there are usually players who seem to be irreplaceable. When they leave,
+the board is often struggling to find a proper replacement for them. That's why there are more
+transfers on those positions. Let's find out on which positions in season
+2015/2016 FC Barcelona spent money.
+
+```opencypher 
+MATCH (n:Team )-[:TRANSFERRED_FROM ]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player ),
+(s:Season)<-[:HAPPENED_IN]-(t)-[r1:TRANSFERRED_TO]->(m:Team)
+WHERE t.fee IS NOT NULL AND (s.name IN ["2015/2016","2016/2017"]) AND m.name="FC Barcelona"
+RETURN collect( p.name) AS names, p.position AS position, SUM(t.fee) AS money_spent_per_position
+```
+
+9) But what was the most money FC Barcelona spent per position from 1992/1993 season to 2019/2020?
+
+```opencypher 
+MATCH (a:Team )-[:TRANSFERRED_FROM ]->(t:Transfer)<-[:TRANSFERRED_IN ]-(p:Player ), (t)-[:TRANSFERRED_TO]->(b:Team)
+WHERE t.fee IS NOT NULL AND b.name="FC Barcelona"
+WITH max(t.fee) as maxMoneySpent, p.position AS position
+RETURN maxMoneySpent, position
+```
+
+10) Now, let us find who were the most expensive players per position in our team FC Barcelona. 
```

Comment:
This looks more like `most expensive player positions` rather than `most expensive players per position`.

Reply:
:+1: